hoprd/rest-api/src/session.rs

```diff
@@ -671,10 +717,12 @@ pub(crate) struct SessionCloseClientRequest {
 )]
 pub(crate) async fn close_client(
     State(state): State<Arc<InternalState>>,
+    // Query(SessionCloseClientRequest { listening_ip, port }): Query<SessionCloseClientRequest>,
     Path(protocol): Path<IpProtocol>,
-    Json(SessionCloseClientRequest { listening_ip, port }): Json<SessionCloseClientRequest>,
+    Path(ip): Path<String>,
```

Comment:
`Path` can be specified only once - this will throw an error when trying to reach the API endpoint.
You need to create the `SessionCloseClientQuery` object that contains `protocol`, `ip` and `port` fields and then use that as `Path(close_req): Path<SessionCloseClientQuery>`.